demo/ARContainerViewController.swift

```diff
@@ -10,12 +10,10 @@ final class ARContainerViewController: UIViewController {
 
     private lazy var mapViewController = ARMapNavigationController()
     private lazy var arViewController = VisionARViewController()
-    
-    weak var navigationDelegate: NavigationManagerDelegate? {
-        didSet {
-            arViewController.navigationDelegate = navigationDelegate
-        }
-    }
+
+    weak var navigationDelegate: NavigationManagerDelegate?
```

Comment:
Please rename it to `navigationManagerDelegate`. That’s a bad practice to have such confusing names.

Reply:
Renamed delegate to `NavigationDelegate`